lib/cretonne/meta/gen_settings.py

```diff
@@ -87,6 +87,9 @@ def gen_getters(sgrp, fmt):
     """
     fmt.doc_comment("User-defined settings.")
     with fmt.indented('impl Flags {', '}'):
+        fmt.doc_comment('Returns inner slice of bytes.')
+        with fmt.indented('pub fn bytes(&self) -> &[u8] {', '}'):
+            fmt.line('&self.bytes')
```

Comment:
The `bytes` array contains a number of non-boolean settings at the front that we don't want included here. Notice how the `numbered_predicate` function below adds `self.boolean_offset` to the byte array offset.

Two changes here:

- Rename `bytes()` to `predicate_bytes()` to make it clear that the byte-sized settings are not included.
- Return the `&self.bytes[boolean_offset]` sub-slice instead of the full array.